cloud-controller-manager/do/resources.go

```diff
@@ -348,8 +261,13 @@ func (r *ResourcesController) syncTags() error {
 			continue
 		}
 
-		name := getDefaultLoadBalancerName(svc)
-		if id, ok := loadBalancers[name]; ok {
+		id := getLoadBalancerID(svc)
+		if id == "" {
+			name := getDefaultLoadBalancerName(svc)
+			id = loadBalancerIDsByName[name]
+		}
+
+		if id != "" {
```

Comment:
You could either do an `else` here or just put a `continue` at the end of the `if id == "" {}` block.

Reply:
We cannot assume that we have an ID for sure at this point: for the case that the user renamed the LB, `loadBalancerIDsByName[name]` won't yield anything. I added a comment to make that clearer.

Reply:
Gotcha.  Yeah, now I see this.  Sorry for the unnecessary comment?

Reply:
Not at all -- it pointed out that the reasoning behind this wasn't clear.